### PR TITLE
Adding SDK for ServiceBus

### PR DESCRIPTION
**Pull request by sazeesha**

### CODE REVIEW

**Comment by sazeesha** [comment]
@markcowl Can you please review this as well. I had split the PR and this is for ServiceBus. I have addressed your comments from Eventhub here.

**Comment by sazeesha** [comment]
@markcowl

File: src/ResourceManagement/ServiceBus/ServiceBus.Tests/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]
```
Comment (shahabhijeet) [inline src/ResourceManagement/ServiceBus/ServiceBus.Tests/Properties/AssemblyInfo.cs]: Remove commented code

**Comment by shahabhijeet** [comment]
:shipit:

**Pull request merged**